recipes/crc_cpp/all/conanfile.py

```diff
@@ -38,16 +40,16 @@ def _supported_compiler(self):
 
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
-            tools.build.check_min_cppstd(self, "17")
+            check_min_cppstd(self, "17")
         if not self._supported_compiler:
             raise ConanInvalidConfiguration("crc_cpp: Unsupported compiler: {}-{} "
                                             "Minimum C++17 constexpr features required.".format(self.settings.compiler, self.settings.compiler.version))
     def source(self):
-       tools.files.get(self, **self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+        get(self, **self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
 
     def package(self):
         self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
```

Comment:
You will need to use full paths like dst=os.path.join(self.package_folder, "licenses") in Conan 2.x.